pkg/util/eventutil/events.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
How about moving this to the package `pkg/operator/k8sutil` instead of creating a new package?

Reply:
Done

Reply:
Done